videoframes.py

```diff
@@ -209,6 +216,16 @@ def close_rewinded(self, event):
         # Let the garbage collector do its magic
         self.rewinded = None
 
+    def fullscreen_rewinded(self, _):
```

Comment:
I'm guessing that Qt passes something to this method but you don't want to use the argument?